### ISSUE TITLE
Rayquaza eggs are glitched and 3 cannot be obtained. 

### ISSUE THREAD

**Issue by SystemErrorCA** [SmashMC-Development/Bugs-and-Issues#1733]
**Describe the Bug**
On rayquaza for the easter event three of the eggs are not allowing players to collect them so they cannot finish the event.

**To Reproduce**
1. Go to ray
2. Collect the eggs
3. notice three are not obtainable

**Servers with the Bug**
Rayquaza

**Expected behavior**
expected to be able to finish the event.

**Screenshots**
![image](https://user-images.githubusercontent.com/120758795/230854238-0763dbe1-fb0e-4559-95c4-85fc3ea02452.png)

![image](https://user-images.githubusercontent.com/120758795/230854317-ba06263b-5289-4808-917a-9eed8a22c222.png)

**Additional context**
Add any other context about the problem here. If nothing, please add 'N/A'.

**Comment by IceSlimGuys** [comment]
Correction:

There is actually 49 eggs available at Rayquaza Spawn. I (and other people) was able to collect all of them. Some think that the eggs that keep their particles even after clicking on it don't give any gift but that's just a misunderstanding. They got their rewards the first time around but I guess for the last one they were just looking for eggs with particles but had the nasty surprise thinking it was unclaimed loot, except it's not...

The last one is most likely supposed to be in the location shown in the Screenshot above, except it's not actually there...
Staff told me that there was no issues with the egg, and that everything was where it should be. But after countless hours looking for it, no one on Rayquaza was able to find it. 

Then, 3 possibilities : 
- We are all dumb
- The hiding spot is GodTier
- It just doesn't exist

Here is the location of every single egg (49) and my location on that screen matches the location of the previous screenshot 
![image](https://user-images.githubusercontent.com/109310452/231004641-a1c6e24e-c628-4efe-98b5-6d6dd7bf5563.png)

If you know exactly where is each single egg please verify the location of every single one of them using my map and also fix remaining particle effect so people won't get fooled by them anymore

Thank you !

**Comment by Superfire96** [comment]
So is one of the eggs missing?

**Comment by IceSlimGuys** [comment]
tldr Yes

**Comment by Archie4690** [comment]
I added in another one the other day, and I have just added in one more, so there should be 53 placed, but if 3 aren't working then we are Gucci and swag now
